anitya/db/models.py

```diff
@@ -345,6 +345,9 @@ class Project(Base):
     latest_version = sa.Column(sa.String(50))
     logs = sa.Column(sa.Text)
 
+    last_check = sa.Column(sa.DateTime, default=datetime.datetime.utcnow, index=True)
+    next_check = sa.Column(sa.DateTime, default=datetime.datetime.utcnow, index=True)
```

Comment:
I looked at the [SQLAlchemy docs](https://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.TIMESTAMP) and there's a timestamp type that supports time zones. I'd recommend just using that.

Reply:
I will use it than.

Reply:
After playing with it for some time, I found out that this isn't working with PostgreSQL.
See https://stackoverflow.com/questions/414952/sqlalchemy-datetime-timezone
Not sure how we should handle the timezone awareness than.

Reply:
That question isn't about the data type I linked so I'm not sure what you mean by "isn't working".

Reply:
Maybe I wasn't clear enough, I tried to use that data type with `timezone=True` and the information is still saved to database without timezone information.
So every time I try to compare it with timezone aware datetime I get error.

Reply:
Does "that data type" mean DateTime? If so that's not surprising, the documentation says "For timezone support, use at least the TIMESTAMP datatype, if not the dialect-specific datatype object." which is a subclass of DateTime.

Reply:
No, I tried to use the TIMESTAMP datatype with `timezone=True`.

Reply:
@jeremycline 
Any other suggestion about timezones? Otherwise it looks good as it is.

Reply:
Apologies, I've not had much time to devote to this. Did you remember to drop the database and re-create it? In any case, since you're the one on the hook for problems if you're comfortable with the current approach, then I'll approve it.

Reply:
It's possible I didn't remember to recreate database, I will try it again.

Reply:
I recreated whole vagrant and I'm still getting this error:
```
TypeError: can't compare offset-naive and offset-aware datetimes
```
The database doesn't store offset, even when you have it defined like this:
```
    last_check = sa.Column(sa.TIMESTAMP(timezone=True),
                           default=arrow.utcnow().datetime, index=True)
    next_check = sa.Column(sa.TIMESTAMP(timezone=True),
                           default=arrow.utcnow().datetime, index=True)
```
Only solution to this, I see, is to store every datetime value to database in UTC.